website/client/components/chat/copyAsTodoModal.vue

```diff
@@ -36,9 +36,18 @@ export default {
       task: {},
     };
   },
+  generateURL () {
+    if (this.group.Type === 'party') {
+      return `${message.user || 'system message'}${message.user ? ' wrote' : ''} in [${this.groupName}](${baseURL}/party)`;
```

Comment:
`${message.user || 'system message'}${message.user ? ' wrote' : ''} in [${this.groupName}]` is three times in one method, I'd suggest to extract it to a local variable and then use that variable for each of the urls